src/main/java/zd/zero/waifu/motivator/plugin/motivation/BaseMotivation.kt

```diff
@@ -64,14 +66,22 @@ abstract class BaseMotivation(
                     )
             )
 
-    override fun onAlertClosed(notification: Notification) {
+    override fun closeNotification() {
+        if (::notification.isInitialized && isNotificationShowing) {
+            if (isNotificationShowing) notification.expire()
+            stopSoundPlayer()
+            notifyDisposal()
+        }
+    }
+
+    override fun isNotificationShowing(): Boolean = notification.isExpired.not()
```

Comment:
What was the idea behind making `isNotificationShowing` part of the public API?
If it should remain public, it should probably also have the initialization check so it does not do the big sad in the future.

Reply:
@zaerald Is this something that you where going to take care of, or did I miss something?

Reply:
I'll work on this for my next update.